tests/models/test_pairwise_probit.py

```diff
@@ -505,7 +508,10 @@ def test_1d_pairwise_server(self):
 
         for _i in range(n_init + n_opt):
             next_config = ask(server)
-            next_y = bernoulli.rvs(f_pairwise(f_1d, next_config["x"], noise_scale=0.1))
+        
+            next_x = torch.tensor(next_config["x"], dtype=torch.float32)
```

Comment:
Must it be a float32? We are trying to make everything float64 (in fact the default tensor float will be float64 soon).

Reply:
Must be a mistake, I will change that